### DISCUSSION TITLE
I am a student of remote education, and I have been rejected for uploading my information and tuition receipt on the Xuexin website

### DISCUSSION THREAD

**Discussion by zhupeikang** [community/community#59501] [GitHub Education]
### Hi everyone,

I am a student of remote education, and I have been rejected for uploading my information and tuition receipt on the Xuexin website

**Comment by zhupeikang** [comment]
I have been rejected more than 40 times. Because I am a distance learner, my positioning is far from my school.

**Reply by HAQ-NAWAZ-MALIK** [reply]
Same brother, i am facing the same issue while submitting my application .
I am also finding solutions.

**Comment by zhupeikang** [comment]
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio. The verification process looks at your GitHub profile to cross reference the evidence submitted. Do you use a variation of your name or a nickname? Is your name in non-english? When you update your profile information please log out and log back into GitHub before re-applying.


Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

**Comment by zhupeikang** [comment]
我放弃了。已经申请了块50次了。学信网证明。学费发票，课程证明。成绩证明，录取通知书都申请过了，但就是不通过，我不能理解
